SpywareFilter/sections/specific.txt

```diff
@@ -52,6 +52,7 @@
 !
 ||cc.loginfra.com/cc?a=$image
 ||cc.naver.com/cc$image,other,ping
+||api.eduki.com/api/v2/events/track
```

Comment:
`v2` is not necessary.
`||api.eduki.com/api/*/events/track` or `||api.eduki.com/*/track`

Reply:
And cookie is still not blocked

Reply:
Yep, what about this rule?
```
eduki.com##.wrap > div.modal-bg
```